contracts/ToucanRegenBridge.sol

```diff
@@ -90,13 +85,21 @@ contract ToucanRegenBridge is Ownable, Pausable, AccessControl {
     //      Constructor
     // ----------------------------------------
 
-    constructor(address tokenIssuer_, INCTPool nctPool_) Ownable() {
-        tokenIssuer = tokenIssuer_;
+    constructor(
+        INCTPool nctPool_,
+        bytes32[] memory roles,
+        address[] memory accounts
+    ) {
+        require(accounts.length == roles.length, "accounts and roles must have same length");
         nctPool = nctPool_;
+        _grantRole(DEFAULT_ADMIN_ROLE, msg.sender);
         _grantRole(PAUSER_ROLE, msg.sender);
-        if (tokenIssuer_ != address(0)) {
-            _grantRole(PAUSER_ROLE, tokenIssuer_);
-            emit TokenIssuerUpdated(address(0), tokenIssuer_);
+        for (uint256 i = 0; i < accounts.length; ++i) {
+            require(
+                roles[i] == TOKEN_ISSUER_ROLE || roles[i] == PAUSER_ROLE,
```

Comment:
We can avoid assigning roles to `msg.sender` since post-deployments all of the roles need to be revoked and avoid restricting `DEFAULT_ADMN_ROLE` here. Actually this check should be removed altogether and we should just ensure there is a single admin assigned.

Reply:
Hmm, similar to what Rodrigo has done in https://github.com/ToucanProtocol/tokenizer/pull/2234/files#diff-c68e1476ab8eae3300691c9701dbb9ac5fdaddc28023a8753e63ef3728a80f3f, but in this case, we want to ensure there is **just one** admin.